src/main.js

```diff
@@ -52,33 +60,52 @@ function displayCurrentCover() {
 
 function displayFormView() {
   homeViewSection.classList.add("hidden");
-  savedViewSection.classList.add("hidden"); 
-  formViewSection.classList.remove("hidden");  
-  randomCoverButton.classList.add("hidden"); 
+  savedViewSection.classList.add("hidden");
+  formViewSection.classList.remove("hidden");
+  randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
   viewSavedButton.classList.remove("hidden");
+  makeYourOwnButton.classList.add("hidden");
+  //hide makeyourownbutton
 }
-   
+
 function displaySavedView() {
-  savedViewSection.classList.remove("hidden"); 
+  savedViewSection.classList.remove("hidden");
   homeViewSection.classList.add("hidden");
-  formViewSection.classList.add("hidden");  
-  randomCoverButton.classList.add("hidden"); 
+  formViewSection.classList.add("hidden");
+  randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
-  viewSavedButton.classList.add("hidden"); 
+  viewSavedButton.classList.add("hidden");
+  makeYourOwnButton.classList.remove("hidden");
 }
 
 function displayHomeView() {
-  homeViewSection.classList.remove("hidden"); 
-  formViewSection.classList.add("hidden");  
-  savedViewSection.classList.add("hidden"); 
-  homeButton.classList.add("hidden"); 
-  randomCoverButton.classList.remove("hidden"); 
+  homeViewSection.classList.remove("hidden");
+  formViewSection.classList.add("hidden");
+  savedViewSection.classList.add("hidden");
+  homeButton.classList.add("hidden");
+  randomCoverButton.classList.remove("hidden");
   saveCoverButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
   viewSavedButton.classList.remove("hidden");
-}
\ No newline at end of file
+  makeYourOwnButton.classList.remove("hidden");
+}
+
+function collectUserInput() {
+  event.preventDefault();
+  covers.push(userImageInput.value);
+  titles.push(userTitleInput.value);
+  descriptors.push(userDescriptor1.value, userDescriptor2.value);
```

Comment:
this is what my PR comment is referring to... we still met Iteration 3 requirements but should fix this.